source/ProtonPack/Header.h

```diff
@@ -298,6 +298,13 @@ bool b_switch_mode_override = false; // Year mode override flag controlled by th
 bool b_pack_on = false;
 bool b_pack_shutting_down = false;
 
+/*
+ * Function prototypes for handling sound effects.
+*/
+void playEffect(int i_track_id, bool b_track_loop = false, int8_t i_track_volume = i_volume, bool b_fade_in = false, unsigned int i_fade_time = 0);
```

Comment:
I've just noticed that both headers do not include a prototype for the stopEffect function, but I assume that should be added?

Reply:
Oops good change, I will add it.